qcodes/utils/time.py

```diff
@@ -0,0 +1,38 @@
+"""
+This module provides helpers for time measurements
+"""
+
+from time import perf_counter
+
+from qcodes.instrument.parameter import Parameter
+
+
+class ElapsedTimeParameter(Parameter):
+    """
+    Parameter to measure elapsed time. Measures wall clock time since the
+    last reset of the instance's clock. The clock is reset upon creation of the
+    instance. The constructor passes kwargs along to the Parameter constructor.
+
+    Args:
+        name: the local name of the parameter. Should be a valid
+            identifier, ie no spaces or special characters. If this parameter
+            is part of an Instrument or Station, this is how it will be
+            referenced from that parent, ie ``instrument.name`` or
+            ``instrument.parameters[name]``
```

Comment:
is it really needed to explain this in the docstring of this parameter? this applies to all parameters, right?

Reply:
Correct. It has been shortened now.